base/repl/REPL.jl

```diff
@@ -1057,7 +1062,7 @@ function run_frontend(repl::StreamREPL, backend::REPLBackendRef)
     dopushdisplay && popdisplay(d)
 end
 
-function start_repl_server(port)
+function start_repl_server(port::Int)
```

Comment:
It's an internal function and if the value is too large, this won't work anyway, so Int is ok.